kong/db/strategies/postgres/init.lua

```diff
@@ -483,22 +479,60 @@ local function execute(strategy, statement_name, attributes, is_update)
 
   clear_tab(argv)
 
-  for i = 1, argc do
-    local name  = argn[i]
-    local value
+  local update = options and options.update
+  local ttl = dao.schema.ttl
+  local ttl_value
+
+  if ttl then
+    ttl_value = options and options.ttl
+    if ttl_value then
+      if ttl_value == 0 then
+        ttl_value = escape_literal(connector, null, fields.ttl)
+
+      elseif update and attributes.created_at and
+             fields.created_at                and
+             fields.created_at.timestamp      and
+             fields.created_at.auto then
+        ttl_value = escape_literal(connector, ttl_value + attributes.created_at, fields.ttl)
 
-    if i == argc and is_update and attributes[UNIQUE] then
-      value = attributes[UNIQUE]
+      elseif update and attributes.updated_at and
+             fields.updated_at                and
+             fields.updated_at.timestamp      and
+             fields.updated_at.auto then
+        ttl_value = escape_literal(connector, ttl_value + attributes.updated_at, fields.ttl)
+
+      else
```

Comment:
Should this be `elseif not update then`? Not sure, but we'd rather avoid touching the TTL if we are updating a row without editing its `created_at` field, right? Unless I missed something that already prevents that, or that makes this behaviour desirable.

Reply:
We won't touch ttl If not requested on updates.